mod_ci/controllers.py

```diff
@@ -369,22 +369,26 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
 
         test_branch.checkout(True)
 
+        gh = GitHub(access_token=g.github['bot_token'])
+        repository = gh.repos(g.github['repository_owner'])(g.github['repository'])
```

Comment:
You cannot use the `g` global context here, because this method can also be called outside Flask (with the cron).

Reply:
Oh, I'll remedy